src/core/graph.js

```diff
@@ -169,80 +169,76 @@ export class Graph<NP, EP> {
   }
 
   /**
-   * Gets the array of all out-edges from the node at the given address.
-   * The order of the resulting array is unspecified.
+   * Find the neighborhood of a given nodeAddress
```

Comment:
revenge of https://github.com/sourcecred/sourcecred/pull/162#discussion_r184813987

Reply:
:)